shell/edit/workload/types/Deployment.vue

```diff
@@ -178,10 +191,18 @@ export default {
             </template>
           </Tabbed>
         </Tab>
-        <Tab :label="'Deployments'" :name="'deployments'" :weight="99">
-          <Labels v-model="value" :mode="mode" />
+        <Tab :label="'Deployment'" :name="'deployments'" :weight="99">
```

Comment:
We should avoid using hardcoded strings `:label`. On line 196 `label-key` is used instead, which looks like the right pattern. 🙏